### DISCUSSION TITLE
我们可以考虑将一些指标参数，提供一些汉化、讲解

### DISCUSSION THREAD

**Discussion by zanglikun** [apache/hertzbeat#499] [Ideas]
<img width="674" alt="image" src="https://user-images.githubusercontent.com/61591648/206943404-9ed7b198-466e-45db-a083-de2b009172b3.png">

**Comment by zanglikun** [comment]
我不太擅长前端。所以我暂时没能力实现，只能提供想法💡

**Reply by tomsun28** [reply]
可以可以有这个👍 就是会有使用者对指标有疑惑 目前只能在帮助文档看指标详解

**Comment by tomsun28** [comment]
已经支持 https://github.com/dromara/hertzbeat/issues/1212
